### PR TITLE
Make video storage directory configurable

### PR DESCRIPTION
**Pull request by duxovni**

### CODE REVIEW

File: musicautils.py [hunk 1]
```diff
@@ -1,12 +1,10 @@
 import os
 import re
 
-DATA_DIR = os.path.join(os.getenv("HOME"), "musicazoo_videos")
-
```
Comment (celskeggs) [inline musicautils.py]: Can we just assign DATA_DIR in here, rather than replicating the logic to get the new value?

**Comment by duxovni** [comment]
I'm personally happier having all the configurable values in one discoverable location that can be edited in a non-version-controlled file. If I need to edit the code to change the data dir, I'll have to stash and pop when I pull new changes down.

File: musicautils.py (hunk 1 above)
Comment (duxovni) [reply musicautils.py]: I'm personally happier having all the configurable values in one discoverable location that can be edited in a non-version-controlled file. If I need to edit the code to change the data dir, I'll have to stash and pop when I pull new changes down.

Comment (celskeggs) [reply musicautils.py]: No, I mean -- the code `os.getenv("MZ_DATA_DIR")` is now duplicated. I'm fine having MZ_DATA_DIR be from the in the configuration file.

**Pull request merged**